frontend/src/pages/profile/styles.module.scss

```diff
@@ -0,0 +1,37 @@
+.profile {
+  width: 1591px;
+  margin: 0 auto;
+}
+
+.userInfoBlock {
+  display: flex;
+  flex-direction: column;
+  width: 270px;
+}
+
+.avatar {
+  position: relative;
+  display: block;
+  width: 270px;
+  height: 269px;
+  overflow: hidden;
+  border-radius: 50%;
+}
+
+.userInfo {
+  margin: 20px 0;
+  font-size: 32px;
+  line-height: 38px;
+}
+
+.userInfo > * {
+  margin: 10px;
+}
```

Comment:
Make it with `gap` in `userInfo`

Reply:
I need to make margin smaller, that's why I can't use gap here.
But I noticed that I can use gap in .userInfoBlock and delete margin in .userInfo